lib/rules/selector-not-notation/index.mjs

```diff
@@ -79,46 +81,109 @@ const rule = (primary, _, context) => {
 
 			if (!selectorRoot) return;
 
+			/** @param {Pseudo} not */
+			const fixSimple = (not) => {
```

Comment:
[suggest] Can we keep the original position of `fixSimple()` to minimize diff?

Reply:
No we can't unless we pass tons of arguments to the function.
i.e. it needs to be in the right scope

see also https://github.com/stylelint/stylelint/pull/7755#discussion_r1637024503

Reply:
Lemme try something.

Reply:
done